packages/@aws-cdk/aws-appsync/README.md

```diff
@@ -636,3 +636,24 @@ api.addMutation('addFilm', new appsync.ResolvableField({
 ```
 
 To learn more about top level operations, check out the docs [here](https://docs.aws.amazon.com/appsync/latest/devguide/graphql-overview.html). 
+
+#### Subscription
+
+Every schema **can** have a top level Subscription type. The top level `Subscription` Type
+is the only exposed type that users can access to invoke a response to a mutation. This means
+you can make any data source real time by specificy a GraphQL Schema directive on a mutation. 
+
+**Note**: The AWS AppSync client SDK automatically handles subscription connection management.
```

Comment:
i think its nice to have something here because subscriptions are unique to AppSync. Thus for a new person closing over the README, it would be nice to get a gloss of how that would work.